container/Containerfile

```diff
@@ -13,8 +13,11 @@ FROM alpine
 RUN apk add python3
 RUN addgroup --system ovirtimg
 RUN adduser --system --ingroup ovirtimg ovirtimg
-RUN mkdir /run/ovirt-imageio /etc/ovirt-imageio
+RUN mkdir /run/ovirt-imageio /etc/ovirt-imageio /images
 COPY --from=build /venv /venv
 COPY conf.d /etc/ovirt-imageio/conf.d
+COPY file.json file.json
+COPY disk.img /images
+RUN chown ovirtimg:ovirtimg /images/disk.img
```

Comment:
This assumes that the images is always called disk.img, which is not a good assumption. The file name and location inside the container should be dynamic. The ticket will point to the location of the file inside the container.

Onwership and permissions should be set by the user of the container so imageio can access the image. This should not be part of the container file.